pkg/rsyslog/init.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+mkdir -p /run/watchdog/pid 2>/dev/null || :
```

Comment:
Does this need the `|| :`? Doesn't the `mkdir -p` return `0`, whether the directory existed or not? Of course, if it cannot make the directory and it does not exist, then it will return an error, but it probably should error in that case.

Reply:
Good catch -- fixed